src/backend/distributed/citus--5.1-3--5.1-4.sql

```diff
@@ -0,0 +1,6 @@
+CREATE OR REPLACE FUNCTION worker_apply_shard_ddl_command(bigint, text, text)
```

Comment:
Could you use ALTER FUNCTION?

This does not replace the existing worker_apply_shard_ddl_command() because argument count is different.